chemprop/uncertainty/evaluator.py

```diff
@@ -1,93 +1,156 @@
-from abc import abstractmethod
+from abc import ABC, abstractmethod
 
 from torch import Tensor
 
 from chemprop.utils.registry import ClassRegistry
 
 
-class UncertaintyEvaluator:
-    """
-    A class for evaluating the effectiveness of uncertainty estimates with metrics.
-    """
+class EvaluatorBase(ABC):
```

Comment:
does this class really provide anything for the abstraction beyond a signature?

Reply:
The original thought is to use this to make the registry `UncertaintyEvaluatorRegistry = ClassRegistry[EvaluatorBase]()`, but I think I can just use `UncertaintyEvaluatorRegistry = ClassRegistry[]()` here.